posthog/demo.py

```diff
@@ -235,3 +229,22 @@ def demo(request):
             create_anonymous_users_ch(team=team, base_url=request.build_absolute_uri("/demo"))
 
     return render_template("demo.html", request=request, context={"api_token": team.api_token})
+
+
+def create_demo_team(organization: Organization, user: User, request: Request) -> Team:
+    team = Team.objects.create_with_data(
+        organization=organization, name=TEAM_NAME, ingested_event=True, completed_snippet_onboarding=True, is_demo=True,
```

Comment:
Worth renaming the demo name, i.e. remove the "Hogflix" maybe?

Reply:
Not in this PR as it would cause issues with the migration.